config/prod.json

```diff
@@ -6,6 +6,7 @@
   "catalogUrlRoot": "https://catalog.dsde-prod.broadinstitute.org",
   "dataRepoUrlRoot": "https://data.terra.bio",
   "dockstoreUrlRoot": "https://dockstore.org",
+  "drsHubUrlRoot": "https://us-central1-broad-dsde-prod.cloudfunctions.net",
```

Comment:
After the change in approach in https://github.com/DataBiosphere/terra-ui/pull/3247/commits/9778ef7f4a207cdaff083048d50db845c3d16bb3, should non-dev configurations have DRS Hub URLs instead of Martha URLs?

Reply:
Yep great point. Thanks for the thorough review on this - sorry about the back and forth, I'm a bit rusty on the UI.

Reply:
No cause to apologize! Back and forth is just the review process working.